CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ Changes since v2.4
 - API for creating catalogue item and its localizations has been changed.
   There is now a single API call that is used to create both a catalogue
   item and the localizations, namely, /api/catalogue-items/create.
+- API for editing workflow content has been changed. The API endpoint for
+  editing is now /api/workflows/edit, whereas the API endpoint for changing
+  the state of a workflow remains /api/workflows/update.
```

Comment:
Hmm,
a) if you're going to make a breaking change, why not rename /update to /state while you're at it
b) if you add new /edit and /state endpoints, you can retain the current /update for backwards compatibility (it'll just call edit-workflow and set-workflow-state)

Reply:
This new version now has /edit, /archived, and /enabled endpoints, as we seemed to have a reasonable consensus about that already, even though ADR is still in review.